File: vane-portals/src/main/java/org/oddlama/vane/portals/menu/PortalMenuTag.java
package org.oddlama.vane.portals.menu;

import java.util.UUID;

public class PortalMenuTag {

    private final UUID portalId;

    public PortalMenuTag(final UUID portalId) {
        this.portalId = portalId;
    }

    public UUID portalId() {
        return portalId;
    }
}

File: vane-portals/src/main/kotlin/org/oddlama/vane/portals/menu/PortalMenuTag.kt
package org.oddlama.vane.portals.menu

import java.util.*

/**
 * Stores the portal id associated with an opened portal menu.
 *
 * @property portalId the tagged portal id.
 */
class PortalMenuTag(private val portalId: UUID?) {
    /** Returns the tagged portal id. */
    fun portalId() = portalId
}
